cos-fleet-catalog-connectors-it/aws/aws-it/src/test/groovy/org/bf2/cos/connector/camel/it/ConnectorIT.groovy

```diff
@@ -447,6 +447,8 @@ class ConnectorIT extends KafkaConnectorSpec {
                           uriEndpointOverride: ${aws.endpoint}
                           overrideEndpoint: true
                     steps:
+                      - to: kamelet:cos-encoder-bytearray-action
+                      - to: log:test?showAll=true&multiline=true
```

Comment:
that is actually intended, we added camel-log to the connector images just because is could be useful. And this test check that the component is there. 
And it is in the tests code so...

Reply:
Ah, i didnt realise its within the test itself ;)